lib/rules/require-shorthand-arrow-functions.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Require shorthand arrow function expressions when returning a single statement.
+ *
+ * Type: `Boolean`
+ *
+ * Value: `true`
+ *
+ * Version: `ES6`
+ *
+ * #### Example
+ *
+ * ```js
+ * "requireShorthandArrowFunctions": true
+ * ```
+ *
+ * ##### Valid
+ *
+ * ```js
+ * evens.map(v => v + 1);
+ * evens.map(v => {
+ *     v = v + 1;
+ *     return v;
+ * });
+ * ```
+ *
+ * ##### Invalid
+ *
+ * ```js
+ * evens.map(v => { return v + 1; });
```

Comment:
That is not a "single statement", which is hard to get your head around it, i'd say this rule is strange, is there any use-cases for it?